src/Verify.NUnit.Tests/VerifyInTestSetUp.IsVerified.verified.txt

```diff
@@ -0,0 +1 @@
+﻿I can call verify in a SetUp method, and the test fixture and test method is available for the verify file name.
```

Comment:
if the verify fails, shouldnt this file not exist?

Reply:
Verify does not fail when called in a Setup method. 
This test is to document that this functionality exists.

In a Setup method Verify is able to identify a test name to use for the verified file, 
because the setup method is invoked as part of a test method.

The issue in identifying the test name occurs when calling verify in a Onetime Setup method.
since OneTimeSetup happens before any tests are run.

There is no .verified.txt file for the VerifyInOneTimeSetup class - since Verify fails in that context.
The test for One time setup asserts that a meaningful exception message is given to the user.